metagraph/src/graph/representation/base/sequence_graph.cpp

```diff
@@ -113,7 +113,8 @@ void call_sequences_from(const DeBruijnGraph &graph,
                          ProgressBar &progress_bar,
                          bool call_unitigs,
                          uint64_t min_tip_size,
-                         bool kmers_in_single_form) {
+                         bool kmers_in_single_form,
+                         bool select_first_edge) {
```

Comment:
my point is not to change first to last (either is fine, it's equivalent), but to make this a default and remove `bool select_first_edge`

Reply:
As discussed last night, we can't just remove `select_first/last_edge` without affecting the length of the contigs. If we make it a default, we'll be shortening contigs that could be otherwise continued. Am I missing something?

Reply:
I see now. Ok, in this case, let's copy the entire function, so instead of changing all the interfaces we add a new method to boss.cpp.

It's fine that the code is copied, this method is suboptimal anyway and the function will be removed as soon as we figure out the best way of assigning anchor nodes.

Reply:
I just don't like to further complicate the interfaces for the sake of a new suboptimal feature, which will be changed in the future.

Reply:
also, then there will be no confusion about the other graph representations, it will be immediately clear that they don't support this annotation (again, will be fixed in the future).